flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -1030,6 +1030,32 @@ void createTemporarySystemFunction(
      */
     boolean dropTemporaryTable(String path);
 
+    /**
+     * Drops a table registered in the given path.
+     *
+     * <p>Temporary objects can shadow permanent ones. If a temporary object exists in a given path,
+     * make sure to drop the temporary object first using {@link #dropTemporaryTable}. This method
+     * can only drop permanent objects.
+     *
+     * <p>Compared to SQL, this method will not throw an error if the table does not exist. Use
```

Comment:
I am wondering if we could change `Compared to SQL`  to `Compared to ANSI SQL` which I guess is what we mean here. As Flink SQL is SQL as well.

Reply:
No here we mean Flink SQL and we compare Table API vs Flink SQL (usually called just SQL) behavior